docs/rules/v-if-else-key.md

```diff
@@ -4,11 +4,12 @@ sidebarDepth: 0
 title: vue/v-if-else-key
 description: require key attribute for conditionally rendered repeated components
 ---
+
 # vue/v-if-else-key
 
 > require key attribute for conditionally rendered repeated components
 
-- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> **_This rule has not been released yet._** </badge>
```

Comment:
I am not adding that underscore, for some reason when I hit save the asterisk is converting to an underscore 🙄

Reply:
that is a weird formatting setting, I did not add it (for what?!), so I wonder why this was not an issue the first time that I submitted 🤷
Anyhow, solved it